saas/models.py

```diff
@@ -1528,6 +1528,26 @@ def provider(invoiced_items):
             result = get_current_provider()
         return result
 
+    def by_organization(self, organization, transaction_type):
```

Comment:
This is really more ``Transaction`` movements from the view point of a specific account for a specific organization.

It might be good to refactor the functionality into ``by_organization(self, organization)`` and ``by_account(self, account)`` such that ``Transaction.objects.by_organization(organization).by_account(account)`` returns the same ``QuerySet``.